example/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.10)
+
+add_executable(hasher hasher.cpp)
+target_include_directories(hasher
+    PRIVATE $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>)
```

Comment:
We can use cmake's interface library mechanism here. Specifying the library `pico` with `target_link_libraries` automatically specify `target_include_directries` related to `pico` to `hasher` target.
```suggestion
target_link_libraries(hasher PRIVATE pico)
```